piecewise_web/index.html

```diff
@@ -239,7 +240,7 @@ <h3 class="league-gothic">Thank you!</h3>
 
 	function getNdtServer () {
 		var xhr = new XMLHttpRequest(),
-			mlabNsUrl = 'https://mlab-ns.appspot.com/ndt?format=json';
+			mlabNsUrl = 'https://mlab-ns.appspot.com/ndt_ssl?format=json';
```

Comment:
You did this correctly. You'll noticed in the output of the query to `/ndt_ssl` that the `fqdn` item has the dashed domain name (e.g., ndt-iupui-mlab1-den04.measurement-lab.org) as opposed to the dotted one, which is necessary for our wildcard certificate to work on the *.measurement-lab.org pattern.